src/locale/locales/de/messages.po

```diff
@@ -257,43 +248,43 @@ msgstr "Geburtstag:"
 #: src/view/com/profile/ProfileHeader.tsx:282
 #: src/view/com/profile/ProfileHeader.tsx:389
 msgid "Block Account"
-msgstr "Konto sperren"
+msgstr ""
 
 #: src/view/screens/ProfileList.tsx:522
 msgid "Block accounts"
-msgstr "Kontos sperren"
+msgstr ""
 
 #: src/view/screens/ProfileList.tsx:472
 msgid "Block list"
-msgstr " Liste sperren"
+msgstr "Blockliste"
 
 #: src/view/screens/ProfileList.tsx:307
 msgid "Block these accounts?"
-msgstr "Diese Kontos sperren?"
+msgstr "Diese Konten blockieren?"
 
 #: src/view/screens/Moderation.tsx:123
 msgid "Blocked accounts"
-msgstr "Gesperrte Konten"
+msgstr "Blockierte Konten"
 
 #: src/view/screens/ModerationBlockedAccounts.tsx:107
 msgid "Blocked Accounts"
-msgstr "Gesperrte Konten"
+msgstr "Blockierte Konten"
 
 #: src/view/com/profile/ProfileHeader.tsx:284
 msgid "Blocked accounts cannot reply in your threads, mention you, or otherwise interact with you."
-msgstr "Gesperrte Konten können nicht in Ihren Beiträgen antworten, Sie erwähnen oder anderweitig mit Ihnen interagieren."
+msgstr "Blockierte Konten können nicht in deinen Threads antworten, dich erwähnen oder anderweitig mit dir interagieren."
```

Comment:
"Threads" may not be so well known to people who are not internet-savvy. The specific 1:1 translation is "Faden" ("Gesprächsfaden"), but more comprehensible is "Beitragsstrang", here the plural "Beitragsstränge", so
"Blockierte Konten können nicht in deinen Beitragssträngen antworten, dich erwähnen oder anderweitig mit dir interagieren."

Reply:
"Beitragsstrang" in my opinion is quite unusual (at least in de_DE). "Threads" might be even better comprehensible, but "Gesprächsfaden" probably the most common word. See also [above](https://github.com/bluesky-social/social-app/pull/2319#discussion_r1437880273)

Reply:
Both "Beitragsstränge" and "Gesprächsfaden" sound super weird and probably wouldn't be understood by people who don't understand "Threads". May I suggest something simple like "Gespräche" or maybe something with "...verlauf"?

Reply:
Again, keep it simple: "Blockierte Konten können dir nicht antworten, dich nicht erwähnen oder anderweitig mit dir interagieren."